### PR TITLE
Refactor project package to allow linkage

### PR DESCRIPTION
**Pull request by sourishkrout**
No change to functionality whatsover.

### CODE REVIEW

**Review by pastuxso — APPROVED** [review]
Looks good to me. :rocket:

File: internal/cmd/tui.go [hunk 1]
```diff
@@ -37,7 +37,8 @@ func tuiCmd() *cobra.Command {
 				return err
 			}
 
-			blocks, err := loadTasks(proj, cmd.OutOrStdout(), cmd.InOrStdin(), false)
+			loader := getLoader(cmd)
```
Comment (pastuxso) [inline internal/cmd/tui.go:40]: It would be fine if `project.NewLoader` would have a return with `error` as well.

Comment (sourishkrout) [reply internal/cmd/tui.go:40]: @pastuxso can you push a commit on top with the changes you have in mind?

**Comment by sourishkrout** [comment]
Thanks @pastuxso!

**Pull request merged**